src/clients/kdump_proposal.rb

```diff
@@ -41,7 +41,24 @@ def main
 
       if @func == "MakeProposal"
         Kdump.Propose
-        @ret = { "raw_proposal" => Kdump.Summary }
+
+        @ret = {
+          "raw_proposal" => Kdump.Summary,
+          # FATE#317488 When expectation at the end of proposal does not match
+          # the value, this proposal will be called again
+          "trigger"      => {
+            "expect"     => "
+              Yast.import \"Kdump\"
+              Yast::Kdump.free_space_for_dump
+            ",
+            "value"      => Yast::Kdump.free_space_for_dump
+          }
+        }
+
+        warning = Kdump.proposal_warnig
+        unless warning.empty?
```

Comment:
IMHO this is bad practice, similar to `description += item unless item.empty?`. Adding an empty object works as intended so we don't need the condition.